PlatformTools/Tools/bootstrapper/bootstrapper.py

```diff
@@ -0,0 +1,317 @@
+#!/usr/bin/env python3

```

Comment:
Because of the dos line endings, the shebang command needs to be
`#!/usr/bin/env -S python3 #`, or when you run it, you get this:

```
» ./bootstrapper.py
/usr/bin/env: ‘python3\r’: No such file or directory
```

Reply:
I agree!
done with https://github.com/openSIL/EDKII-Platform/pull/4/commits/2533c56e980b933009dcb1e882d8610fc6b5275c